db/common_test.go

```diff
@@ -0,0 +1,170 @@
+// Package db - common test
+// The following file contains all the common functions for testing.
+package db
+
+import (
+	"context"
+	"crypto/rand"
+	"database/sql"
+	"fmt"
+	"testing"
+
+	"github.com/packethost/pkg/log"
+)
+
+func newCTX() ctx {
+	c, closer := context.WithCancel(context.Background())
+	return ctx{
+		context: c,
+		cancel:  closer,
+	}
+}
+
+func (a *testArgs) setCTX() {
+	a.ctx = newCTX()
+}
+
+func (a *testArgs) setLogger() {
+	var err error
+	if a.logger, err = log.Init("github.com/tinkerbell/tink"); err != nil {
+		panic(err)
+	}
+}
+
+func (a *testArgs) setPsqlInfo() {
+	a.psqlInfo = fmt.Sprintf("host=%s port=%d user=%s "+
+		"password=%s dbname=%s sslmode=disable",
+		a.host, a.port, a.user, a.password, a.dbname)
+	a.logger.Info(a.psqlInfo)
+}
+
+func makeUUID() (r string) {
+	b := make([]byte, 16)
+	if _, err := rand.Read(b); err != nil {
+		panic(err)
+	}
+	uuid := fmt.Sprintf("%x-%x-%x-%x-%x",
+		b[0:4], b[4:6], b[6:8], b[8:10], b[10:])
+	return uuid
+}
+
+func newTestParams() *testArgs {
```

Comment:
it would be great if this function could generate some random data bits, maybe uuids and some randomish name for the template?